### DISCUSSION TITLE
V1.2.3 SV04 Firmware Updates

### DISCUSSION THREAD

**Discussion by johncarlson21** [johncarlson21/SV04-IDEX-3D-Printer-Mainboard-Source-code#9] [Announcements]
**Updates/Additions:**

- Add PID Tuning to firmware and screen ( currently tests at 200 hotend and 60 bed )
- Remove Multi Probe ( fails with High Speed probing on )
- Upped DEFAULT_MAX_FEEDRATE ( { 500, 500, 20, 25, 25 } to { 500, 500, 20, 50, 50 } )
- Upped default retraction acceleration to 800 and default travel acceleration to 1000
- Enabled quick home to allow the axis's to move faster to home position
- Enabled BLTOUCH_HS_MODE
- Fixed Mesh View display to show 7x7 grid numbers and display the proper probing number
- Upped manual feedrate speeds ( used in the move axis screen )
- Fixed Z height display on print. ( used to show the previous layer height instead of current actual height )
- Put back screen version display ( needs to be updated in printer firmware when screen is updated )
- Updated version number to 1.2.3

UPDATE SCREEN FIRMWARE FOR THIS FIRMWARE TO V1.2.2
[Screen Firmware](https://github.com/johncarlson21/SV04-Touchscreen/releases)

<hr /><em>This discussion was created from the release <a href='https://github.com/johncarlson21/SV04-IDEX-3D-Printer-Mainboard-Source-code/releases/tag/V1.2.3'>V1.2.3 SV04 Firmware Updates</a>.</em>

**Comment by sdevine** [comment]
Thanks for updating and keeping the firmware active.  I have only had my SV04 for a month and it's a great printer when dialed in and having more features via your firmware is super.  Where is the best place to suggestion changes / ideas?  Also - the screen firmware still says 1.2.1 from the link given..

**Comment by johncarlson21** [comment]
New printer release is up 1.2.3, you can see the link on the release notes for the screenJohnOn Jan 7, 2023, at 12:25 PM, Scott Devine ***@***.***> wrote:﻿
Thanks for updating and keeping the firmware active.  I have only had my SV04 for a month and it's a great printer when dialed in and having more features via your firmware is super.  Where is the best place to suggestion changes / ideas?  Also - the screen firmware still says 1.2.1 from the link given..
